script/percy-tests.js

```diff
@@ -1,38 +1,81 @@
-const PercyScript = require('@percy/script')
+const expect = require('expect');
```

Comment:
Lots of this is from the example code for using percy/cli and puppeteer: https://github.com/percy/example-percy-puppeteer/blob/master/tests/todomvc_spec.js#L1